components/formats-common/src/loci/common/LogbackTools.java

```diff
@@ -51,31 +51,52 @@
 
   // -- Constructor --
 
+  public static final String CALLER = "Bio-Formats";
```

Comment:
As mentioned elsewhere, I'd tend to make a field like this private unless there's a reason we want `LogbackTools.CALLER` as part of our public API. As this stands, changing or removing the variable would require a deprecation cycle.